test/integration/auth.spec.ts

```diff
@@ -1652,6 +1655,58 @@ describe('admin.auth', () => {
     });
 
     const fixtures: UserImportTest[] = [
+      {
+        name: 'HMAC_MD5',
+        importOptions: {
+          hash: {
+            algorithm: 'HMAC_MD5',
+            key: Buffer.from('secret'),
+          },
+        } as any,
+        computeSaltFirstPasswordHash: (userImportTest: UserImportTest): Buffer => {
+          return userImportTest.computePasswordHash(userImportTest, 'SALT_FIRST');
```

Comment:
I think this is better done as an option to `importOptions`. After all, this is where this parameter is expected:
```
      importOptions: {
          hash: {
            algorithm: 'HMAC_MD5',
            key: Buffer.from('secret'),
            inputOrder: 'SALT_FIRST',
          },
        } as any,
```

You can then check the order in `computePasswordHash`